docs/HowTo/Configure/Consensus-Protocols/Clique.md

```diff
@@ -7,21 +7,23 @@ source: rinkeby.json
 
 # Clique
```

Comment:
> ./docs/HowTo/Configure/Consensus-Protocols/Clique.md:1: MD041 First line in file should be a top level header
To prevent this you have to convert file header html style comment to dashes syntax:

```
---
title: the page title
description: the page description
---

# Your first level title
```

Reply:
Done.

Reply:
Acronyms and Definitions section added to the end of the file, and [vanity data] definition included there.